reference/5.1/Microsoft.PowerShell.Core/About/about_Remote_Troubleshooting.md

```diff
@@ -662,44 +674,43 @@ contains unsigned script files and formatting files.
 
 To import the modules that are created by these cmdlets, either by using
 `Import-PSSession` or `Import-Module`, the execution policy in the current
-session cannot be Restricted or AllSigned. (For information about Windows
-PowerShell execution policies, see about_Execution_Policies.
+session cannot be Restricted or AllSigned. For information about PowerShell
+execution policies, see
+[about_Execution_Policies](about_Execution_Policies.md).
 
 To import the modules without changing the execution policy for the local
-computer that is set in the registry, use the Scope parameter of
+computer that is set in the registry, use the **Scope** parameter of
 `Set-ExecutionPolicy` to set a less restrictive execution policy for a single
 process.
 
-For example, the following command starts a process with the RemoteSigned
-execution policy. The execution policy change affects only the current
-process and does not change the PowerShell ExecutionPolicy registry
-setting.
+For example, the following command starts a process with the `RemoteSigned`
+execution policy. The execution policy change affects only the current process
+and does not change the PowerShell **ExecutionPolicy** registry setting.
 
 ```powershell
 Set-ExecutionPolicy -Scope process -ExecutionPolicy RemoteSigned
 ```
 
-You can also use the ExecutionPolicy parameter of **PowerShell.exe** to start
-a single session with a less restrictive execution policy.
+You can also use the **ExecutionPolicy** parameter of **PowerShell.exe** to start a
```

Comment:
Shouldn't PowerShell.exe be in backticks?

Reply:
fixed